internal/httpcontroller/handlers/settings.go

```diff
@@ -79,12 +79,6 @@ func (h *Handlers) SaveSettings(c echo.Context) error {
 		}
 	}
 
-	// Send success notification for reloading settings
-	h.SSE.SendNotification(Notification{
-		Message: "Settings applied successfully",
-		Type:    "success",
-	})
-
```

Comment:
This is merely a proposal: would having only single notification when saving settings improve on UX.